hudi-common/src/test/java/org/apache/hudi/avro/TestHoodieAvroUtils.java

```diff
@@ -253,7 +254,13 @@ public void testRemoveFields() {
     assertEquals("key1", rec1.get("_row_key"));
     assertEquals("val1", rec1.get("non_pii_col"));
     assertEquals(3.5, rec1.get("timestamp"));
-    assertNull(rec1.get("pii_col"));
+    Object removed;
```

Comment:
maybe test by the different avro version, like:
```
if (HoodieAvroUtils.gteqAvro1_10()) {
  // assert to throw AvroRuntimeException.
  // some codes...
} else {
    assertNull(rec1.get("pii_col"));
}
```

Reply:
done